### PR TITLE
Increased max chat message length from 500 characters to 2000 characters

### LINKED ISSUE #367: Chat allows 500 characters
I'm okay with it allowing only 500 characters. However, there should be an indication on how many characters are remaining as someone is typing.

### PR DESCRIPTION
**Pull request by blackshadowshade**
Fixes #367.

Requires database update 00367_chat_length.sql.

http://jenkins.buttonweavers.com:8080/job/buttonmen-blackshadowshade/576/

### CODE REVIEW

File: deploy/database/schema.game.sql [hunk 1]
```diff
@@ -84,10 +84,10 @@ CREATE TABLE game_chat_log (
     game_id            MEDIUMINT UNSIGNED NOT NULL,
     chat_time          TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
     chatting_player    SMALLINT UNSIGNED NOT NULL,
-    -- A chat message is limited to 500 Unicode characters (see
+    -- A chat message is limited to 2000 Unicode characters (see
     -- GAME_CHAT_MAX_LENGTH in BMInterface and Game.js). Since a Unicode
```
Comment (cgolubi1) [inline deploy/database/schema.game.sql]: If GAME_CHAT_MAX_LENGTH was actually moved from BMInterface to ApiSpec, which it seems like based on what you changed in this pull, this comment should be updated to say so.

**Comment by cgolubi1** [comment]
This seems simple.  I'll try it out by updating a database with an open game with some chat in it, and modifying an existing message to be more than 500 chars.  I'd like to see that comment updated per the above, but don't see anything else complicated here.

**Comment by blackshadowshade** [comment]
Comment updated.

**Comment by cgolubi1** [comment]
Looks good.

**Pull request merged**